src/main/java/com/smartcodeltd/jenkinsci/plugins/buildmonitor/Config.java

```diff
@@ -0,0 +1,37 @@
+package com.smartcodeltd.jenkinsci.plugins.buildmonitor;
+
+import com.smartcodeltd.jenkinsci.plugins.buildmonitor.order.ByName;
+import hudson.model.AbstractProject;
+import org.apache.commons.lang3.builder.ToStringBuilder;
+
+import java.util.Comparator;
+
+public class Config {
+
+  private Comparator<AbstractProject> order;
+
+  public Comparator<AbstractProject> getOrder() {
+    return order;
```

Comment:
Is it worth returning a copy of the order object rather than the object itself to avoid possible mutation of state other than by using setOrder and resetting the whole object?